docs/UserGuide.md

```diff
@@ -2,1012 +2,1171 @@
-    - [Task Page Commands](#task-page-commands)
-    - [Grades Page Commands](#grades-page-commands)
-- [Glossary](#glossary)
+* Please see the [Glossary](#glossary) section for more detailed and specific definitions.
 
---------------------------------------------------------------------------------------------------------------
-
-## Quick Start
-
-1. Ensure you have Java 11 or above installed in your Computer.
-2. Download the latest clipboard.jar from [here](https://github.com/AY2223S2-CS2103T-T15-4/tp/releases).
-3. Copy the file to the folder you want to use as the *home folder* for your CLIpboard.
-4. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar clipboard.jar` command to run the application.&nbsp;
-<br>e.g. your `clipboard.jar` is stored in the `user/app/task/` directory. You run `cd user/app/task/`, then `java -jar clipboard.jar`.
-<br>A GUI similar to [<i>Figure 1</i>](#introduction) should appear in a few seconds.
-<br><i>Note how the app contains some sample data.</i>
-5. You will start on the `Course page` (Refer to [<i>Figure 3</i>](#navigation-guide) below for the Navigation Guide).
-6. Type your command in the command box (Refer to [<i>Figure 2</i>](#visual-guide) below for the Visual Guide) and press Enter/Return to execute it.
-<br>e.g. typing `help` and pressing the Enter/Return key will open the help window.
-    <br>Some example commands you can try:
+---
+
+## Getting started
+
+### Setting up
+#### Installation
+1. Ensure you have Java 11 or above installed on your Computer.
+2. Download the latest `clipboard.jar` from [here](https://github.com/AY2223S2-CS2103T-T15-4/tp/releases).
+3. Copy `clipboard.jar` to the folder you want to use as the home folder of CLIpboard.
+
+#### Launching CLIpboard
+*Option 1*:
+* Double-click on `clipboard.jar` to run it. If this does not work, go to *Option 2*.
+
+*Option 2*:
+1. Open a command terminal. [Guide to open a command terminal](https://www.ionos.com/help/email/troubleshooting-mail-basicmail-business/access-the-command-prompt-or-terminal/#:~:text=Open%20Command%20Prompt%20in%20Windows,cmd%22%20and%20then%20click%20OK.).
+2. Copy the file path of the **folder** you put `clipboard.jar` in.
+    * [Guide For Windows](https://www.howtogeek.com/670447/how-to-copy-the-full-path-of-a-file-on-windows-10/)
+    * [Guide For MacOS](https://support.apple.com/en-sg/guide/mac-help/mchlp1774/mac#:~:text=Show%20the%20path%20to%20a%20file%20or%20folder&text=Choose%20View%20%3E%20Show%20Path%20Bar,bottom%20of%20the%20Finder%20window.)
+    * [Guide For Linux](https://www.tutorialspoint.com/get-the-full-path-of-a-file-in-linux)
+3. In the command terminal, type `cd <FILE_PATH>`, where `<FILE_PATH>` is the file path you found in step 2.
+4. Type `java -jar clipboard.jar` into the command terminal and press `Enter` to launch CLIpboard.&nbsp;
+
+If you see something like this, you're good to go!<br>
+<img src="./images/Ui.png" width="650" height="529"/>
```

Comment:
This image looks better with the colours to show in our UG, but would it be better to show the `UiCoursePage.png` instead? Since it is what they would see when they first launch it.

Reply:
good catch. that was my intention but chose wrong file